indico/modules/rb/client/js/common/bookings/BookingExportModal.jsx

```diff
@@ -70,8 +69,26 @@ export default function BookingExportModal({rooms, onClose}) {
               <FinalRadio name="format" value="csv" label={Translate.string('CSV')} />
               <FinalRadio name="format" value="xlsx" label={Translate.string('XLSX')} />
             </Form.Group>
-            <FinalDatePeriod name="dates" label={Translate.string('Period')} required allowNull />
-            <FinalRoomSelector name="rooms" label={Translate.string('Rooms to export')} required />
+            <FinalDatePeriod
+              name="dates"
+              label={Translate.string('Period')}
+              validate={v.dates()}
+              disabledDate={() => false}
+              required
+              allowNull
+            />
+            <FinalRoomSelector
+              name="rooms"
+              label={Translate.string('Rooms to export')}
+              validate={val => {
+                if (!val || !val.length) {
+                  return Translate.string(
+                    'Please choose at least one room to export its bookings.'
```

Comment:
"to export its bookings" sounds a bit weird; i think "Please choose at least one room" is clear enough